content/en/docs/components/pipelines/v2/introduction.md

```diff
@@ -7,45 +7,47 @@ weight = 1
 
 {{% stable-status %}}
 
-Kubeflow Pipelines (KFP) is a platform for building and deploying portable, 
-scalable machine learning workflows using Docker containers.
+Kubeflow Pipelines (KFP) is a platform for building and deploying portable and
+scalable machine learning (ML) workflows by using Docker containers.
 
-To get started quickly with a KFP deployment and usage example, see the [Quickstart][quickstart] guide.
+KFP is available as a core component of Kubeflow or as a standalone installation. To quickly get started with a KFP deployment and usage example, see the [Quickstart][quickstart] guide.
+
+<!-- REVIEWER COMMENT (REMOVE BEFORE PUBLISHING): Best to comment these out until the topic is available -->
+<!-- [Learn more about installing Kubeflow][Installation]
+[Learn more about installing Kubeflow Pipelines standalone][Installation] -->
 
 ## Objectives
-Kubeflow Pipelines' primary objectives are to enable:
-* End-to-end orchestration of machine learning workflows
+
+The primary objectives of Kubeflow Pipelines are to enable the following:
+* End-to-end orchestration of ML workflows
 * Pipeline composability through reusable components and pipelines
-* Easy management, tracking, and visualization of pipeline definitions, pipeline runs, experiments, and machine learning artifacts
+* Easy management, tracking, and visualization of pipeline definitions, pipeline runs, experiments, and ML artifacts
 * Efficient use of compute resources by eliminating redundant executions through [caching][caching]
 * Cross-platform pipeline portability through a platform-neutral [IR YAML pipeline definition][ir-yaml]
 
-KFP is available as a core component of Kubeflow or as a standalone installation.
```

Comment:
Suggested a reorg here, since this does not seem to be directly related to the objectives. We could, perhaps, revisit this after the topic is available?